### DISCUSSION TITLE
Welcome to MEI_encoding Discussions!

### DISCUSSION THREAD

**Discussion by fujinaga** [DDMAL/MEI_encoding#32] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fujinaga** [comment]
@GabbyHalpin @timothydereuse  will let you know where the code is for Clivis / Podatus issue.

**Reply by GabbyHalpin** [reply]
ok great!

**Reply by GabbyHalpin** [reply]
@timothydereuse  let me know if you would like to jump on a zoom!
